crates/fs/src/fs.rs

```diff
@@ -555,10 +555,18 @@ impl Fs for RealFs {
         Ok(())
     }
 
+    #[cfg(not(target_os = "windows"))]
     async fn canonicalize(&self, path: &Path) -> Result<PathBuf> {
         Ok(smol::fs::canonicalize(path).await?)
     }
 
+    #[cfg(target_os = "windows")]
+    async fn canonicalize(&self, path: &Path) -> Result<PathBuf> {
+        use util::paths::SanitizedPath;
```

Comment:
But that thing is a noop already for non-windows?
So we do not need to do that at all it seems and could have always used `SanitizedPath::from`.

Reply:
Personly, I only want using `dunce::simplified`, using `SanitizedPath` only because it is exists there.